server/engine/src/main/java/org/eclipse/lsp/cobol/cli/command/ListSources.java

```diff
@@ -53,7 +53,11 @@ public class ListSources implements Callable<Integer> {
    */
   public Integer call() throws Exception {
     Gson gson = new GsonBuilder().setPrettyPrinting().create();
-    parent.initProcessorGroupsReader(workspace);
+    try {
+      parent.initProcessorGroupsReader(workspace);
+    } catch (Exception e) {
+      return 2;
```

Comment:
Should be a defined constant with a comprehensive name

Reply:
Implemented in latest commit.